monitoring/dashboards/dashboard_provider.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: 1
+
+# dashboard providers
+# we use a single generic one for now.
```

Comment:
avoid use of "we" as it is ambiguous with respect to the party involved

Reworded proposal:
> Use a single generic provider

Reply:
agreed. good feedback- I will update it.